Makefile

```diff
@@ -27,13 +27,10 @@ continous_test:
 
 cleanup:
 	- docker rm -f eventstore_local
-	- docker rm -f eventstore_local_noauth
 
 eventstore_docker:
-	docker run -d --name eventstore_local -p 2113:2113 -p 1113:1113 eventstore/eventstore
-	docker run -d --name eventstore_local_noauth -p 22113:2113 -p 11113:1113 eventstore/eventstore
+	docker run -d --name eventstore_local -p 2113:2113 -p 1113:1113 eventstore/eventstore:release-5.0.8
```

Comment:
@epoplavskis one question I have is that we're not running Eventstore 5 in Prod yet...so does it make sense for us to use it here?

Reply:
actually i see that bogdan upgraded to eventstore 5 without removing the no auth, so i think it's fine to keep it, just maybe revert your changes to the make file and removal of the test. otherwise the rest should be good.